scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+"""
+A script that takes the compliance dataset (the released version
+of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
+and turns it into a directory bundle of binary and JSON files suitable
+for use by the reference server.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import shutil
+import argparse
+import os
+import sys
+from os import path
+from glob import glob
```

Comment:
I understood that `from os import path` and `from glob import glob` were pretty idiomatic. `glob.glob` doesn't really tell you anything you didn't already know, after all. Note I constrained it to these two well-known, supported cases, and used the default `import x` in all others. If you feel strongly about it, I'll change it, though, although I suspect it will hurt, not help readability.